Table.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Table</title>
+    <style>
+        body{
+            font-family: sans-serif;
```

Comment:
Move the CSS into the separate CSS file.

Reply:
@Neha  Done